application/templates/viewdeed.html

```diff
@@ -34,14 +29,10 @@ <h3 class="heading-medium">Lender</h3>
         </div>
 
         <h3 class="heading-medium">Borrower</h3>
-        <div class="grid-row">
+        <div class="bolden-text">
             {% for borrower in deed_data.deed.borrowers %}
-            <div class="column-third">
-              <p>
-                <b>{{ borrower.forename}} {{ borrower.middle_name}}{% if borrower.middle_name %} {% endif %}{{
-                borrower.surname}}</b><br>
-              </p>
-            </div>
+                {{ borrower.forename}} {{ borrower.middle_name}}{% if borrower.middle_name %} {% endif %}{{
+                borrower.surname}}<br>
```

Comment:
better including the entire thing in a <p>, this should give it natural seperation